website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,102 @@
+## Usage
+
+### When to use
+
+- To show content as needed while keeping the UI organized and visually manageable.
+- To hide long pieces of content with a similar structure.
+
+### When not to use
+
+- To hide basic non-critical information to users. Instead, consider using a [Reveal](/components/reveal).
+- To allow users switch between different content within the same context. Instead, consider using [Tabs](/components/tabs).
+
+### Accordion vs Reveal
+
+While similar in functionality and interaction, the Accordion and Reveal are meant to be used in different scenarios and use cases.
+
+#### When to use the Accordion
+
+- You have multiple sections of content that can be expanded or collapsed.
+
+- Each section of content has its own toggle content, giving users the ability to access specific sections.
+
+- Users need to navigate through different sections of content and selectively expand or collapse sections
+
+- There is a sequential or hierarchical relationship between the different sections.
+
+#### When to use the Reveal
+
+- You have a single piece of content that can be expanded to collapsed to reveal additional details or information.
+
+- The content within the toggle is plain text.
+
+- The content within the reveal is relatively short and simple.
+
+- There is no need for multiple independent sections or hierarchical relationship between different sections.
+
+### Usage examples
+
+!!! Do
+
+Use the Accordion when you need to display content in different sections that have a sequential or hierarchical relationship between them.
+
+![Accordion with basic content](/assets/components/accordion/usage-do.png)
```

Comment:
The images have all different sizes, which results in having the examples showcased all with different apparent sizes, which is weird

here the chevron sizes are different
<img width="173" alt="screenshot_2878" src="https://github.com/hashicorp/design-system/assets/686239/7c9f1dbe-d2c8-49b7-a00d-77db69f831c0">

here the text size is differeng
<img width="921" alt="screenshot_2876" src="https://github.com/hashicorp/design-system/assets/686239/b6b197ba-449a-489c-8c66-e3c4b6759ff6">

a simple trick is to use always the same **width** for the images (leaving white space if necessary) so they maintain the same ratio one in relation to the other (even when the page is resized). the alternative is to always set the width of the images in the markdown, which is much more work to do

Reply:
@cveigt

Reply:
Updated!